src/mock-doc/event.ts

```diff
@@ -55,6 +55,26 @@ export class MockCustomEvent extends MockEvent {
 
 }
 
+export class MockKeyboardEvent extends MockEvent {
+  code: string = null;
```

Comment:
is "null" the default value of this properties in the spec? in a real browser

Reply:
Looked the defaults up on MDN and switched all of those values to use the defaults that were published there.

Reply:
So this one is now defaulting to empty string, etc.